src/lib/Libpython/pbs_python_svr_internal.c

```diff
@@ -5230,8 +5230,8 @@ _pbs_python_event_set(unsigned int hook_event, char *req_user, char *req_host,
 	}
 
 	lval = min_restart_interval;
-	if (server.sv_attr[(int)SRV_ATR_PythonRestartMinInterval].at_flags & ATR_VFLAG_SET)
-		min_restart_interval = server.sv_attr[(int)SRV_ATR_PythonRestartMinInterval].at_val.at_long;
+	if (server.sv_attr[(int)SVR_ATR_PythonRestartMinInterval].at_flags & ATR_VFLAG_SET)
```

Comment:
Optional: do we really need `(int)` type cast? someplace we use, someplace we don't

Reply:
That would be for another PR